src/pages/RoomInvitePage.js

```diff
@@ -74,7 +74,10 @@ function RoomInvitePage(props) {
 
     // Any existing participants and Expensify emails should not be eligible for invitation
     const excludedUsers = useMemo(
-        () => [...PersonalDetailsUtils.getLoginsByAccountIDs(lodashGet(props.report, 'visibleChatMemberAccountIDs', [])), ...CONST.EXPENSIFY_EMAILS],
+        () =>
+            _.map([...PersonalDetailsUtils.getLoginsByAccountIDs(lodashGet(props.report, 'visibleChatMemberAccountIDs', [])), ...CONST.EXPENSIFY_EMAILS], (participant) =>
```

Comment:
Not that, this PR is the same as the [original PR](https://github.com/Expensify/App/pull/31010), except that we use ```visibleChatMemberAccountIDs``` rather than ```participantAccountIDs```